pages/content/amp-dev/documentation/guides-and-tutorials/index.email.html

```diff
@@ -8,114 +8,199 @@
+    <h4 class="ap-o-resources-grid-card-headline">Components</h4>
+    <p class="ap-o-resources-grid-card-copy">The complete documented AMP library and its referencees.</p>
+  </a>
+  <a href="{{g.doc('/content/amp-dev/documentation/examples/index.html', locale=doc.locale).url.path}}" class="ap-o-resources-grid-card">
+    <div class="ap-a-ico ap-o-resources-grid-card-icon">
+      {% do doc.icons.useIcon('/icons/examples-flyout.svg') %}
+      <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#examples-flyout"></use></svg>
     </div>
-
-  </div>
-</div>
-
-<hr>
-
-<div class="ap--benefits--list">
-  <div class="ap-m-benefit-twocolumns">
-
-    <h2>AMP for Email Providers</h2>
-    <p class="ap-m-benefit-text-copy">Bring interactive mail to your users by integrating the AMPHTML Email specification with your platform. Get started by reading the base requirements and get in touch with our team through GitHub.</p>
-
-
-  </div>
-
-  <div class="ap-m-benefit-twocolumns">
-
-  </div>
-</div>
-
-<div class="ap-o-teaser-grid-featured">
-
-  <h2>Featured Documents</h2>
-  <div class="ap-o-teaser-grid">
-    <div class="ap-o-teaser-grid-list">
-      {% with %}
-      {% set teaser_doc = g.doc('/content/amp-dev/documentation/guides-and-tutorials/contribute/integrate-amphtml-email.md', locale=doc.locale) %}
-      {% include '/views/partials/teaser.j2' %}
-      {% endwith %}
-
-      {% with %}
-      {% set teaser_doc = g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-format.md', locale=doc.locale) %}
-      {% include '/views/partials/teaser.j2' %}
-      {% endwith %}
-
-      {% with %}
-      {% set teaser_doc = g.doc('/content/amp-dev/community/platform-and-vendor-partners.html', locale=doc.locale) %}
-      {% include '/views/partials/teaser.j2' %}
-      {% endwith %}
+    <h4 class="ap-o-resources-grid-card-headline">Examples</h4>
+    <p class="ap-o-resources-grid-card-copy">The fastest way to learn is to peak over the tangible examples.</p>
+  </a>
+  <a href="{{g.doc('/content/amp-dev/documentation/templates/index.html', locale=doc.locale).url.path}}" class="ap-o-resources-grid-card">
+    <div class="ap-a-ico ap-o-resources-grid-card-icon">
+      {% do doc.icons.useIcon('/icons/templates.svg') %}
+      <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#template"></use></svg>
     </div>
-  </div>
-
-</div>
-
-<h2>Get Involved</h2>
-<p class="limit">
-  AMP is proudly an open source project with an active community of contributors. We welcome and appreciate documentation and translation contributions! Check out our community section to join our mission to make the web suck less.
-</p>
+    <h4 class="ap-o-resources-grid-card-headline">Templates</h4>
+    <p class="ap-o-resources-grid-card-copy">Don't want to start from scratch? Start with a ready-made design.</p>
+  </a>
+</section>
+
+{% do doc.styles.addCssFile('/css/components/organisms/video-slider.css') %}
```

Comment:
It would be super nice to move this into a partial and then use a similar pattern as with the teaser to use `{% with %}`

Reply:
Done :)